### ISSUE TITLE
Adding 2 SendEmail save actions breaks Sitecore

### ISSUE THREAD

**Issue by alinulms** [bartverdonck/Sitecore-Forms-Extensions#75]
**Describe the bug**
If I add 2 SendEmail actions when I'm saving the form then I'm thrown an error as if Sitecore can't save the form with the same button 2 times. 

**To Reproduce**
1. Go to 'forms'
2. Create a simple form with an Email field and a button
3. Create a Send Email save action and select any message, and as Mail To the email field value
4. Click on 'Apply' and 'Save'
5. Create a second Send Email action and select any message and in the Mail To select a Fixed email address. 
6. Click on 'Apply' and 'Save'

**Expected behavior**
Red error ribbon will appear and show an error. 
**Screenshots**
![image](https://user-images.githubusercontent.com/4048118/74150154-379dfb00-4c01-11ea-96ed-2e72cab4ba1e.png)

**Comment by SchelstraeteRuben** [comment]
Try setting the AllowDuplicateItemNamesOnSameLevel setting to true.

**Comment by bartverdonck** [comment]
Hi, thanks for the report. 

1) Looks like this is introduced with Sitecore 9.3, I did not encounter this in previous versions.
2) After investigation this is not a bug in Sitecore Forms Extentions.  The same behaviour takes place when you try to add 2 out of the box actions as well (e.g. redirect to url).
3) I have tried the workaround suggested by @SchelstraeteRuben . I confirm this workaround to be valid.

Best Regards
